### PR TITLE
added github repo/credits as footer

### PR DESCRIPTION
**Pull request by kirbyfern**
is this subtle enough as footer?

### CODE REVIEW

**Comment by kirbyfern** [comment]
screen shot 2017-04-12 at 5 07 53 pm

**Comment by rickr** [comment]
Can we make it a little smaller and a little less conspicuous? I'd also like a link to this repo in there somewhere.

**Comment by kirbyfern** [comment]
Ok , link to this repo like pull requests are welcome?

**Comment by kboygit** [comment]
screen shot 2017-04-12 at 10 10 29 pm

Update version

File: index.html [hunk 1]
```diff
@@ -130,11 +130,15 @@
...
       </section>
-    <!-- <div class="footer">
+
+<div id="footer">
+    <footer>
+        <span>Built by <a href="http://deployopcode.io/" class="underline">Deploy</a> The Dev Shop for <a href="https://operationcode.org/" class="underline">Operation Code.</a></span>
+    </footer>
     <footer>
-      <span>Built by <a href="http://deployopcode.io/">Deploy</a> The Dev Shop for <a href="https://operationcode.org/">Operation Code</a>
+        <a class="underline" href="https://github.com/OperationCode/vettec">Pull Request are welcome.</a>
```
Comment (hollomancer) [inline index.html]: Change to 'Pull requests are welcome!'

File: index.html [hunk 2]
```diff
@@ -130,11 +130,15 @@
           <a href='https://operationcode.org'><img src="images/medal_navy.png" alt="opcodelogo"></a>
         </div>
       </section>
-    <!-- <div class="footer">
+
+<div id="footer">
+    <footer>
+        <span>Built by <a href="http://deployopcode.io/" class="underline">Deploy</a> The Dev Shop for <a href="https://operationcode.org/" class="underline">Operation Code.</a></span>
```
Comment (hollomancer) [inline index.html]: Change copy to 'Deploy, the Dev Shop'

**Review by hollomancer — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (kirbyfern) [reply index.html]: screen shot 2017-04-13 at 12 34 58 pm

Latest changes

File: index.html (hunk 1 above)
Comment (kirbyfern) [reply index.html]: screen shot 2017-04-13 at 12 34 58 pm

sorry accidentally removed the last comment , here is the latest changes

**Review by hollomancer — APPROVED** [review]

**Pull request merged**